nutanix/resource_nutanix_image.go

```diff
@@ -279,19 +279,10 @@ func resourceNutanixImageCreate(d *schema.ResourceData, meta interface{}) error
 	request.Metadata = metadata
 	request.Spec = spec
 
-	imageUUID, err := resourceNutanixImageExists(conn, n.(string))
```

Comment:
Why did you remove this validation?

Reply:
The API does allow images with the same name so i don't see a reason why the provider should change that.

Should the API change to not allow this the creation will fail anyways.